src/components/ErrorView.js

```diff
@@ -0,0 +1,98 @@
+import { Fragment } from 'react'
+import { div, h, iframe, pre } from 'react-hyperscript-helpers'
+import Collapse from 'src/components/Collapse'
+import * as Style from 'src/libs/style'
+import * as Utils from 'src/libs/utils'
+import { Component } from 'src/libs/wrapped-components'
+
+
+class StackTraceView extends Component {
```

Comment:
None of these track state or lifecycle, so why not just make them functions?

Reply:
I like ErrorView being a component for cosmetic reasons, but I'll convert the other two.